skt/kerneltree.py

```diff
@@ -307,6 +327,15 @@ def merge_git_ref(self, uri, ref="master"):
         return (SKT_SUCCESS, head)
 
     def merge_patchwork_patch(self, uri):
+        """
+        Merge the patch from Patchwork run.
+
+        Args:
+            uri: URL of patch on a Patchwork instance.
+
+        Raises:
+            Exception in case the failed to apply patch
```

Comment:
This should mention the specific `PatchApplicationError` instead of generic `Exception`